### ISSUE TITLE
Goods distribution for mines is not working

### ISSUE THREAD

**Issue by jaschaio** [jsettlers/settlers-remake#688]
Again, I haven't played Settlers for ages so I am not a hundred percent sure if I am right here. But if I remember well than it was best to give fish only to gold mines, meat only to iron mines and bread only to coal mines. 

I have put it like that inside the Goods distribution menu, but still all mines get all kind of food.

<img width="249" alt="screen shot 2017-12-18 at 18 27 34" src="https://user-images.githubusercontent.com/11058666/34119079-26deafe8-e421-11e7-992b-b257d37f2525.png">

**Comment by DerKrawallkeks** [comment]
You're right, it is like that in the original settlers. I think it's not implemented yet.
